client/src/store/usePlayer.ts

```diff
@@ -9,7 +9,7 @@ export function usePlayer(): [string | null, (key: string | null) => void] {
     [gameId],
   );
   const localhostKey = gameId
-    ? `${gameId}${String(localhostKeyFromUrl)}_player`
+    ? `${gameId}${localhostKeyFromUrl ?? ""}_player`
```

Comment:
What's this for?

Reply:
we are never passing nothing in here, so it shouldn't matter.  the String() convesion was to make TS happy I think

Reply:
I have a bunch of keys in my storage that look like `ABCDEnull_player`

Reply:
that isn't better than `ABCD_player?

Reply:
It means we have a bug

Reply:
Oh, wait I am dumb, I forgot what this was